tests/ui_tests/test_cluster_list_page.py

```diff
@@ -1685,6 +1685,31 @@ def check_expectations():
         cluster_config_page.config.click_on_advanced()
         check_expectations()
 
+    def test_two_fields_on_cluster_config_page(self, sdk_client_fs: ADCMClient, app_fs):
+        """Test two different fields on group config page"""
+
+        path = get_data_dir(__file__) + "/cluster_with_two_different_fields"
```

Comment:
you can use just `get_data_dir(__file__, 'cluster_with_two_different_fields')`

Reply:
fixed, thanks)